ui/components/component-library/button-base/button-base.js

```diff
@@ -127,7 +132,7 @@ ButtonBase.propTypes = {
    * The size of the ButtonBase.
    * Possible values could be 'SIZES.AUTO', 'SIZES.SM', 'SIZES.MD', 'SIZES.LG',
```

Comment:
Let's add the pixel height sizes next to the abstracted sizes as per the dev user test today. Pixels are 👑